packages/cashier/src/pages/account-transfer/account-transfer-form/__tests__/account-transfer-form.spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable testing-library/no-container,testing-library/no-node-access */
```

Comment:
Just for my understanding, why you disabled this?

Reply:
It wasn't supposed to be ignored. thanks for mentioning it. fixed.